spec/features/settings/external_cert_feature_spec.rb

```diff
@@ -67,66 +154,66 @@
     end
 
     it "sucessfully uploads velum cert/key" do
-      attach_file("external_certificate_velum_cert", ssl_cert_file_a)
-      attach_file("external_certificate_velum_key", ssl_key_file_a)
+      attach_file("external_certificate_velum_cert", crt_v_a)
+      attach_file("external_certificate_velum_key", key_v_a)
 
       click_button("Save")
       expect(page).to have_http_status(:success)
       expect(page).to have_content(success_message)
     end
 
     it "sucessfully uploads kubeAPI cert/key" do
-      attach_file("external_certificate_kubeapi_cert", ssl_cert_file_a)
-      attach_file("external_certificate_kubeapi_key", ssl_key_file_a)
+      attach_file("external_certificate_kubeapi_cert", crt_k_a)
+      attach_file("external_certificate_kubeapi_key", key_k_a)
 
       click_button("Save")
       expect(page).to have_http_status(:success)
       expect(page).to have_content(success_message)
     end
 
     it "sucessfully uploads dex cert/key" do
-      attach_file("external_certificate_dex_cert", ssl_cert_file_a)
-      attach_file("external_certificate_dex_key", ssl_key_file_a)
+      attach_file("external_certificate_dex_cert", crt_d_a)
+      attach_file("external_certificate_dex_key", key_d_a)
 
       click_button("Save")
       expect(page).to have_http_status(:success)
       expect(page).to have_content(success_message)
     end
 
     it "sucessfully uploads velum, kubeAPI, and dex cert/key" do
-      attach_file("external_certificate_velum_cert", ssl_cert_file_a)
-      attach_file("external_certificate_velum_key", ssl_key_file_a)
-      attach_file("external_certificate_kubeapi_cert", ssl_cert_file_a)
-      attach_file("external_certificate_kubeapi_key", ssl_key_file_a)
-      attach_file("external_certificate_dex_cert", ssl_cert_file_a)
-      attach_file("external_certificate_dex_key", ssl_key_file_a)
+      attach_file("external_certificate_velum_cert", crt_v_a)
+      attach_file("external_certificate_velum_key", key_v_a)
+      attach_file("external_certificate_kubeapi_cert", crt_k_a)
+      attach_file("external_certificate_kubeapi_key", key_k_a)
+      attach_file("external_certificate_dex_cert", crt_d_a)
+      attach_file("external_certificate_dex_key", key_d_a)
 
       click_button("Save")
       expect(page).to have_http_status(:success)
       expect(page).to have_content(success_message)
     end
 
     it "sucessfully lists Subject Alternative Names" do
-      attach_file("external_certificate_velum_cert", ssl_cert_file_b)
-      attach_file("external_certificate_velum_key", ssl_key_file_b)
+      attach_file("external_certificate_velum_cert", crt_v_b)
```

Comment:
What are the difference between set "a" and "b"?  Maybe a comment somewhere so someone regenerating or otherwise following up knows what the intended difference is?
(it's entirely possible that I missed this in another script)

Reply:
Ah, that makes sense, and is definitely a good test to include.

Reply:
Ah, that makes sense, and is definitely a good test to include.